RetrospectiveExtension.Frontend/utilities/tokenHelper.tsx

```diff
@@ -0,0 +1,28 @@
+import { Buffer } from 'buffer';
+
+export interface JwtPayload {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
+  iss?: string | undefined;
+  sub?: string | undefined;
+  aud?: string | string[] | undefined;
+  exp?: number | undefined;
+  nbf?: number | undefined;
+  iat?: number | undefined;
+  jti?: string | undefined;
+}
+
+/**
+ * Decode JSON Web Token.
+ */
+export const decodeJwt = (token: string): null | JwtPayload => {
+  try {
+    const base64Main = token.split('.')[1];
+    const decodedMain = Buffer.from(base64Main, 'base64').toString();
+
+    return JSON.parse(decodedMain);
+  }
+  catch (e) {
+    return null;
```

Comment:
Should we track this exception via app insights?

Reply:
Yes, but I think that we should address it in a different story, something like `Tracking all errors with App Insights`. Right now if you search for `throw new Error` in the code, there are many places that we're doing that and they're good candidates for tracking exceptions with AI.

What do you think? @argoldbe @NarmathaBala

Reply:
Sounds good - maybe just add a TODO for now.

Reply:
Added.